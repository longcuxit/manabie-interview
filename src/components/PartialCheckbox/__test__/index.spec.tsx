import { render, screen } from "@testing-library/react";
import { PartialCheckbox } from "..";

describe("components/PartialCheckbox:", () => {
  it("snapshot checked", async () => {
    const { container } = render(
      <PartialCheckbox checked onChange={Boolean} />
    );
    expect(container).toMatchSnapshot();
  });

  it("should show partial status", async () => {
    const { container } = render(<PartialCheckbox onChange={Boolean} />);
    expect(container.querySelector(".check-partial")).toBeInTheDocument();
  });
});
