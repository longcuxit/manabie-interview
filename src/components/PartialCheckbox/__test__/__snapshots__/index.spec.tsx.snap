// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/PartialCheckbox: snapshot checked 1`] = `
<div>
  <div
    class=""
  >
    <input
      checked=""
      class="form-check-input"
      type="checkbox"
    />
  </div>
</div>
`;
